mobly/suite_runner.py

```diff
@@ -77,12 +77,60 @@ def setup_suite(self, config):
 from mobly import records
 from mobly import signals
 from mobly import test_runner
+from mobly import utils
 
 
 class Error(Exception):
   pass
 
 
+class SuiteInfoRecord:
+  """A record representing the test suite info in test summary.
+
+  Attributes:
+    test_suite_class: str, the class name of the test suite class.
+    test_suite_version: str, the test suite version.
+    begin_time: int, epoch timestamp of when the suite started.
+    end_time: int, epoch timestamp of when the suite ended.
+  """
+
+  KEY_TEST_SUITE_CLASS = 'Test Suite Class'
```

Comment:
still not sure what this means

Reply:
It's the class name of test suite class. Do you mean that `Test Suite Class` is confusing and I'd better use `Test Suite Class Name`?

I didn't use `Test Suite Class Name` just because `records.py` is using `Test Class` for test class name.

Reply:
My preference is still either Suite Class or Suite Class Name. Having `Test` here doesn't really make it clearer, as "suite" in Mobly already unambiguously means "test suite". If the clarification needed is to show that this is the literal Python class name of the suite in question, then maybe Suite Python Class?
<!-- Reviewable comment -OEp1quK9FXW1wgFL-1T:b-v3ikf4 -->
<!-- Sent from Reviewable.io -->

Reply:
Thanks, I'd prefer "Suite Class Name".
<!-- Reviewable comment -OErZtS3CLbBWitF4GSF:bbqtjg5 -->
<!-- Sent from Reviewable.io -->